test-ets.sh

```diff
@@ -14,7 +14,7 @@ final_exit_code=0
 
 function run_and_annotate {
   echo "running retesteth $1"
-  ets/retesteth -t "$1" &> "retesteth-$1-log.txt"
+  timeout 5m ets/retesteth -t "$1" -- --verbosity 3 &> "retesteth-$1-log.txt"
```

Comment:
According to the logs of the build, the ETS tests did run and completed, but the top summary says 
```
retesteth: GeneralStateTests -- --

retesteth crashed; check the artifacts
```
Maybe we need a longer timeout?

Reply:
ETS used to run very quickly (whole suite in a minute or so), I'm kinda curious why it's drastically slower now.